doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -15,6 +15,57 @@
 <a name="1.1"></a>
 ## 1.1 Introduction
 
+This document (at this stage, not a RA, we're just using the template as a placeholder), discusses the role of Kubernetes in managing NFV software components (e.g. VNFCs). Application containers such as those managed by Kubernetes can be considered a Foundational PaaS capability<sup>1</sup> and so the following terms can all be thought of as being the topic of this document:
+- Kubernetes PaaS (Platform-as-a-Service)
+- CaaS (Containers-as-a-Service)
+- Kubernetes-based CaaS (Containers-as-a-Service)
+- Kubernetes-as-a-Service
+
+Kubernetes itself is a “system for automating deployment, scaling, and management of containerized applications” and therefore Kubernetes place within our architecture should therefore be closely linked to the application lifecycle. However, it is very important to point out that Kubernetes Platforms also consist of other open source projects, or add-ons, such as:
+- CNI-compliant network plugins
+- CSI-compliant storage plugins
+- CRI-compliant container runtimes
+- service mesh options
+- service discovery options
+- monitoring and logging options
+- etc.
+
+Therefore, when considering software validation, and in NFV scenarios the compliance, verification and certification complexities, more thought is required around how flexible the "Platform" should be, with regards to these add-ons.  i.e. does a single blueprint for a "Kubernetes Platform for NFV" include specific versions of specific projects for each add-on, meaning that if you change from, e.g. Envoy to Fluentd, a new blueprint is created to certify against.
+
+There are a number of different approaches that could be adopted:
+1. Software vendor brings Kubernetes
+    - This option places the operator purely as the infrastructure provider, and the software vendor uses the VIM, or IaaS APIs, to manage the lifecycle of the infrastructure required to create, update and delete Kubernetes clusters ***and*** the software being deployed into Kubernetes
+2. Operator brings Kubernetes
+    - This option places the operator as not only the infrastructure provider, but also a "platform-as-a-service" provider (for Kubernetes Platform). The operator may already be providing platform-as-a-service offerings such as database-as-a-service, or other underlying "enabler services" for the software such as DNS, NTP, mail relay, etc.
+    - It is not necessarily the case that the vendor that the operator chooses for Kubernetes is the same vendor that provides the NFVI and/or VIM for that operator. That is a procurement decision for each operator.
+
```

Comment:
3rd option Telcom vendor brings K8s , right now all Telco vendor like Huawei , E/// , Cisco has their own Container platform whick based on K8s ( Huawei has Fusion stage ) , E// ( Cloud Container Distribution (CCD) ) and Cisco has Cisco Container Platform 
Which lead us to the same dilemma that shall we ( Operators ) go with Sw vendor such as RH OCP , Pivtol , .. or  go with Telco vendor solution !!!

Reply:
This is the 1st option, the software vendor is the telecom vendor.

Reply:
SW vendors are different than the Telco vendor , we as opertaors need to avoid vendor lock as we did in NFV ( VIM ) that we dont like to have the VIM from the same vendor of VNF

So the SW is CNF/VNF agnostic such as RH , Vmware , ... but the Telco vendor that there own CNF